pkg/server/tunnel.go

```diff
@@ -71,7 +71,7 @@ func (t *Tunnel) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	klog.V(4).InfoS("Set pending", "random", random, "value", w)
 	backend, err := t.Server.BackendManager.Backend(context.TODO())
 	if err != nil {
-		http.Error(w, fmt.Sprintf("currently no tunnels available: %v", err), http.StatusInternalServerError)
+		klog.ErrorS(err, "currently no tunnels available")
```

Comment:
Why the change here? I think it still makes sense to send the error back to the client if we have no tunnels.

Reply:
I believe this is switching to the new structured logging system (https://godoc.org/github.com/kubernetes/klog#ErrorS) and will automatically attach the error in a standardized manner.

Reply:
Right, this makes sense if we we were changing from another form of klog. But going from sending an error on the tunnel back to the client to only logging the error locally seems like a big jump?

Reply:
Doh!
Your completely correct! We probably want both but we definitely should not remove the http.Error.

Reply:
Can we keep both of them? The `klog` can be helpful when debugging on the server-side.

Reply:
Yes

Reply:
> Why the change here? I think it still makes sense to send the error back to the client if we have no tunnels.

ResponseWriter is already hijacked, and i think the object w can not be used as an error responder. so modified to klog.Errors() and log the error info.